packages/editor/src/components/block-navigation/dropdown.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * WordPress dependencies
+ */
+import { IconButton, Dropdown, SVG } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import BlockNavigation from './';
+
+const menuIcon = (
+	<SVG xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" width="20">
+		<path d="M5 5H3v2h2V5zm3 8h11v-2H8v2zm9-8H6v2h11V5zM7 11H5v2h2v-2zm0 8h2v-2H7v2zm3-2v2h11v-2H10z" />
```

Comment:
Can you use `Path` component instead of HTML tag? We probably should refactor all code occurrences and add linter rule for that.

Reply:
>add linter rule for that.

I did the hard work:

https://astexplorer.net/#/gist/f7f5d60c5bbc381708bff5a035cfe1ad/beb956688752e5b3cf616e77c7b153130b3b979b

Reply:
Thanks Andrew, 

<img width="297" alt="screen shot 2018-10-16 at 15 57 52" src="https://user-images.githubusercontent.com/272444/47025911-46cf7e80-d15c-11e8-8bf6-a47f1faf763f.png">

I'll open a separate PR, as there's still some work to do :P

Reply:
Should we also use the `G` cc @gziolo ?

Reply:
What about rect...?